pkg/tasks/update_template_distributions_test.go

```diff
@@ -0,0 +1,4 @@
+package tasks
+
+type UpdateTemplateDistributionsSuite struct {
+}
```

Comment:
I'll write this test after some review because it's all mocking and if anything changes this will definitely change with it.

Reply:
I'm also not sure if these tests have any value, since the vast majority of the code in them is mocking. I imagine anything covered here is covered by the integration test.

Reply:
I mostly agree.  Anything you can pull out to make smaller, and test that would be good (but i only found the one example)

Reply:
Added one test here